curriculum/challenges/chinese/01-responsive-web-design/applied-visual-design/use-the-u-tag-to-underline-text.md

```diff
@@ -23,7 +25,7 @@ dashedName: use-the-u-tag-to-underline-text
 assert($('u').length === 1);
 ```
 
-`u` 标签的内容文本应为 `Ph.D. students`。
+`u` 标签的文本内容应为 `Ph.D students`。
```

Comment:
We have to get this resolved by Crowdin because it effects our QA validation and allows translators and proofreaders to change the text inside `code` tags.

cc: @miyaliu666 @RafaelDavisH

Reply:
Committing the suggestion.

You are correct - I should be able to get a message drafted and shared with the team today.